config/sync/user.role.site_manager.yml

```diff
@@ -50,6 +50,7 @@ permissions:
   - 'create stanford_news content'
   - 'create stanford_page content'
   - 'create stanford_person content'
+  - 'create su_lockup'
```

Comment:
I haven't checked in the UI but what is this entity?

Reply:
This is cruft from previous code. removing